server/src/collection.rs

```diff
@@ -156,4 +152,29 @@ impl Collection {
     pub async fn alias(&self) -> String {
         self.alias.lock().await.clone()
     }
+
+    pub async fn dispatch_items(&self) -> Result<(), Error> {
+        let keyring_items = self.keyring.items().await;
+        let mut items = self.items.lock().await;
+        let object_server = self.manager.lock().await.object_server().clone();
```

Comment:
```suggestion
        let object_server = self.manager.lock().await.object_server();
```
what is the clone needed for?

Reply:
That doesn't look like the correct fix. Instead
```
let service_manager = self.manager.lock().await;
let object_server = service_manager.object_server();
```
should get around the error, no?

Reply:
Yes, very nicely. Thanks!